src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
```

Comment:
Consider providing a more descriptive error message when rejecting the promise. This will help in debugging and understanding the reason for the rejection.